RunStuff Template.m

```diff
@@ -149,112 +149,94 @@
 evtest=Sort[Eigenvalues[build2bMatrix[Infinity,2.]/.{a-> 0.}]]//Chop;
 
 VerificationTest[
-Re[\[CapitalOmega]ch[2.,2 Pi//N,0.,0]]===Re[evtest] 
+Re[\[CapitalOmega]ch[2.,1.,0.,0]]===Re[evtest] 
 ,TestID-> "2 Beam Growth Rate (Real part)"]
 
 VerificationTest[
-Im[\[CapitalOmega]ch[2.,2 Pi//N,0.,0]]=== Im[evtest]
+Im[\[CapitalOmega]ch[2.,1.,0.,0]]=== Im[evtest]
 ,TestID-> "2 Beam Growth Rate (Imaginary part)"]
 
-Print[Re[\[CapitalOmega]ch[2.,2 Pi//N,0.,0]],Re[evtest]];
-Print[Im[\[CapitalOmega]ch[2.,2 Pi//N,0.,0]],Im[evtest]];
+Print[Re[\[CapitalOmega]ch[2.,1.,0.,0]],Re[evtest]];
+Print[Im[\[CapitalOmega]ch[2.,1.,0.,0]],Im[evtest]];
+
 
-(*The hard coded number 2 10 ^21 is the "common factor" chakraborty calls \[Mu] for this matrix *)
 
 
 
 
 (*Check the dispersion relation from Gail's paper*)
-dispersionCheck[data_,\[CapitalOmega]_,k_]:=Module[{\[Phi]0,\[Phi]1,\[CapitalOmega]p,kp,check,Idis,\[Theta],ms},
+dispersionCheck[data_,\[CapitalOmega]_,k_]:=Module[{\[Phi]0,\[Phi]1,\[CapitalOmega]p,kp,check,Idis,\[Theta]},
 \[Theta]=data["mids"];
 (*Defined in Gail's Blue equation 30 and 31 *)
 \[Phi]0=Sum[munits ndensities[data,"xflavor"-> False][[1,i,i]]-munits ndensities[data,"xflavor"-> False][[2,i,i]],{i,1,Length[\[Theta]]}];
 \[Phi]1=Sum[(munits ndensities[data,"xflavor"-> False][[1,i,i]]-munits ndensities[data,"xflavor"-> False][[2,i,i]])\[Theta][[i]],{i,1,Length[\[Theta]]}];
-ms=Table[munits ndensities[data,"xflavor"-> False][[1,i,i]]-munits ndensities[data,"xflavor"-> False][[2,i,i]],{i,1,Length[\[Theta]]}];
+
 (* "Shifted" Eigenvalue and k*)
 \[CapitalOmega]p=N[\[CapitalOmega]-((munits/mp) *data["Yes"] *data["matters"])-\[Phi]0];
 kp=k-\[Phi]1;
 
 (*Definition of I from Gail's equation (41)*)
-Idis[n_]:=Sum[((munits(ndensities[data,"xflavor"-> False][[1]][[i,i]]- ndensities[data,"xflavor"-> False][[2]][[i,i]]))/(\[CapitalOmega]p-(kp \[Theta][[i]])))\[Theta][[i]]^n,{i,1,Length[\[Theta]]}];
+Idis[n_]:= Sum[(( munits(ndensities[data,"xflavor"-> False][[1]][[i,i]]- ndensities[data,"xflavor"-> False][[2]][[i,i]]))/(\[CapitalOmega]p-(kp \[Theta][[i]])))\[Theta][[i]]^n,{i,1,Length[\[Theta]]}]//Chop;
 
 (*The condition is that Equatrion (43), below, should be 0 if the vacuum is*)
 
-check=(Idis[0]+1)(Idis[2]-1)-(Idis[1]* Conjugate[Idis[1]]);
-
-Return[{check,\[Phi]0,\[Phi]1,\[CapitalOmega]p,kp,Idis[0],Idis[1],Idis[2],ms}]
+check=((Idis[0]+1)(Idis[2]-1))-(Idis[1]^2)//Chop;

```

Comment:
again chop

Reply:
Yes; This is to round very small numbers to 0 for the purposes of the verification check.  This chop *should* be here.